### PR TITLE
README: refresh

### PR DESCRIPTION
**Pull request by CleanMachine1**
The README seems to me like it could use some improvements. Here's some of the things which stand out to me as needing changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,34 +53,38 @@ which is supported by the tldr-pages project maintainers:
...
 For a comprehensive list of clients, head over to our [Wiki](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).
 
-## How do I contribute?
+## How do I contribute to tldr-pages?
 
-- Your favourite command isn't covered?
-- You can think of more examples for an existing command?
+All contributions are welcome!
+
+Some ways include to contribute are:
```
Comment (CleanMachine1) [inline README.md:74]:
```suggestion
Some ways to contribute include:
```

File: README.md [hunk 2]
```diff
@@ -53,34 +53,38 @@ which is supported by the tldr-pages project maintainers:
...
-## How do I contribute?
+## How do I contribute to tldr-pages?
 
-- Your favourite command isn't covered?
-- You can think of more examples for an existing command?
+All contributions are welcome!
+
+Some ways to contribute include:
+
+- Adding your favorite a command which isn't covered.
```
Comment (CleanMachine1) [inline README.md:76]: I have opted to use the American English version of the word favorite, however I have no issue with using `favourite`

File: README.md [hunk 3]
```diff
@@ -25,8 +25,8 @@ The **tldr-pages** project is a collection of community-maintained help pages
 for command-line tools, that aims to be a simpler, more approachable complement
 to traditional [man pages](https://en.wikipedia.org/wiki/Man_page).
 
-Maybe you are new to the command-line world? Or just a little rusty?
-Or perhaps you can't always remember the arguments to `lsof`, or `tar`?
+Maybe you are new to the command-line world; or maybe just a little rusty;
+or perhaps you can't always remember the arguments to `lsof`, or `tar`?
```
Comment (CleanMachine1) [inline README.md:29]: We may want to revert this change if we want more emphasis on the questions being asked.

File: README.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ How about:
 ![animated svg of the tldr client displaying the tar command](images/tldr.svg)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX, Linux, macOS, SunOS and Windows command-line tools.
+for the most common UNIX, Linux, macOS, SunOS, Android and Windows command-line tools.
```
Comment (CleanMachine1) [inline README.md:46]: Android seemed to be missing.

**Review by owenvoke — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (waldyrious) [reply README.md:29]: I slightly prefer the questions. With the semicolons, the text seems a bit like it's describing the reader (and therefore assuming their knowledge, or rather, the lack thereof) rather than explicitly making guesses.

File: README.md [hunk 5]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the fully-featured official [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
```
Comment (waldyrious) [inline README.md:56]: I'd actually go ahead and remove "fully-featured" and "official" from the sentence.

File: README.md [hunk 6]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
 
     pip3 install tldr
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
```
Comment (waldyrious) [inline README.md:60]: Just a couple of rewording suggestions. Not a strong stance — I'm happy to discuss adjustments.

```suggestion
Then you will have direct access to simplified, easy-to-read help for commands, such as tar,
```

File: README.md [hunk 7]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
+accessible through typing `tldr tar` instead of the standard `man tar`
 
 However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+out the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the most recent pages and commits.
```
Comment (waldyrious) [inline README.md:65]: Maybe we should mention the web clients which can also be used without installing anything (and are arguably more convenient than a huge document covering all pages). The PDF is likely a better choice for offline consumption, so we might want to highlight that aspect instead.

File: README.md [hunk 8]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
-## How do I contribute?
+## How do I contribute to tldr-pages?
 
-- Your favourite command isn't covered?
-- You can think of more examples for an existing command?
+All contributions are welcome!
+
+Some ways to contribute include:
+
+- Adding your favorite a command which isn't covered.
```
Comment (waldyrious) [inline README.md:77]:
```suggestion
- Adding your favorite command which isn't covered.
```

File: README.md [hunk 9]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
+- Adding more to an existing page.
```
Comment (waldyrious) [inline README.md:78]:
```suggestion
- Adding examples or improving the content of an existing page.
```

File: README.md (hunk 6 above)
Comment (CleanMachine1) [reply README.md:60]: I agree with the changes other than the will, let's see what others think

File: README.md [hunk 10]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
+- Translating pages into different languages.
 
 All `tldr` pages are kept as Markdown files right here in this repository,
 so you can edit them directly and submit your changes as pull requests.
 
-All contributions are welcome!
 We strive to maintain a [welcoming and collaborative](GOVERNANCE.md) community.
-Have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
+To begin, have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
```
Comment (waldyrious) [inline README.md:85]: I disagree with this change as it is. If you start by saying "To begin", my expectation as a reader is that a series of steps will follow, but no others are offered here. I think the original phrasing works. Can you explain what prompted you to make this change?

File: README.md [hunk 11]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
 
-If you'd like to contribute to translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
-to see the current progress of all translations.
+For translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
```
Comment (waldyrious) [inline README.md:87]: I prefer the original wording. Why are you proposing this change?

File: README.md [hunk 12]
```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
...
+to see the current progress of all translations to see which pages need to be translated.
```
Comment (waldyrious) [inline README.md:88]:
```suggestion
to see the overall progress of all translations, and which translations are missing or outdated.
```

File: README.md (hunk 11 above)
Comment (CleanMachine1) [reply README.md:87]: Are you referring to line 88 as well?

I assume only 87.

From the changes made above, I think it fits better with the changed content specifically new line 79 where translations as a possible option has been listed, so the subject has already been stated and now this paragraph just continues where it was left off.

File: README.md (hunk 10 above)
Comment (CleanMachine1) [reply README.md:85]: I can see your point there, how would you feel about changing it to something like `When starting out...`

File: README.md (hunk 7 above)
Comment (CleanMachine1) [reply README.md:65]: Agreed, I will make a full suggestion tomorrow.

File: README.md (hunk 3 above)
Comment (SethFalco) [reply README.md:29]:
```suggestion
Maybe you're new to the command-line world? Perhaps you're just a little rusty or can't always recall the arguments for commands like `lsof`, or `tar`?
```

Thoughts on this?

* I think starting sentences with a connective is always questionable. I wouldn't go as far as to say invalid, but nice to avoid.
* Using maybe/perhaps (basically synonyms) 3 times in a bit uncomfortable to read.
* Usually a sentence (in English) should have a verb and a noun, a sentence like "Or just a little rusty?" doesn't meet this, so it doesn't seem like a valid sentence.
* Users that are new to command line or don't know what `lsof` or `tar` might not be able to infer that those are examples of commands.
* Use of apostrophes, just for the sake of reducing the word count a bit. It doesn't do much but hey I tried. ^-^'

All changes are subjective, and you can ofc pick and choose which points you agree with or not.

File: README.md (hunk 7 above)
Comment (SethFalco) [reply README.md:65]:
```suggestion
which is always up-to-date with the latest pages and commits.
```

* "most recent" might be a bit wordy in this case, thoughts?

File: README.md (hunk 5 above)
Comment (CleanMachine1) [reply README.md:56]:
```suggestion
Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
```

File: README.md [hunk 13]
```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
...
+accessible through typing `tldr tar` instead of the standard `man tar`
 
-However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+However, if you just want to browse without an internet connection or installing software,
+checkout the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the latest pages and commits.
+
+Also, for browsing without installing a client to your computer,
+see our [web client](https://tldr.ostera.io/)
```
Comment (CleanMachine1) [inline README.md:69]: @waldyrious do you approve of these lines?

File: README.md (hunk 7 above)
Comment (CleanMachine1) [reply README.md]: @SethFalco your suggestion here has been applied.

File: README.md [hunk 14]
```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
```
Comment (EmilySeville7cfg) [inline README.md:54]:
~~~suggestion
```sh
npm install -g tldr
```
~~~

File: README.md [hunk 15]
```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
...
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
     pip3 install tldr
```
Comment (EmilySeville7cfg) [inline README.md:58]:
~~~suggestion
```sh
pip3 install tldr
```
~~~

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (waldyrious) [reply README.md:60]: No strong feelings about the "will". If you would prefer not to include it, I'd be OK with that.

File: README.md (hunk 10 above)
Comment (waldyrious) [reply README.md:88]: Sure, but the "when" feels a bit off to me. I would prefer "If starting out..." (or something along those lines, e.g. "If it's your first time contributing to this project...").

File: README.md (hunk 3 above)
Comment (waldyrious) [reply README.md:29]: I'm OK with Seth's proposal :+1:. All points of the reasoning are sensible.

File: README.md (hunk 13 above)
Comment (waldyrious) [reply README.md:69]: I'd remove the "However", and the "Also". The sentences work well without them. 

I'd also remove the "which is always up-to-date with the latest pages and commits", which is somewhat superfluous information, and is a little at odds with the premise of not needing an Internet connection :)

File: README.md [hunk 16]
```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
...
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
-However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+However, if you just want to browse without an internet connection or installing software,
+checkout the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the latest pages and commits.
+
+Also, for browsing without installing a client to your computer,
```
Comment (waldyrious) [inline README.md:67]: Perhaps "browsing the pages", just to err on the side of explicitness?

And maybe we could expand to "without downloading any content or installing a client...", which links this sentence to the PDF option mentioned above, and covers the case of simply cloning the git repository.

File: README.md [hunk 17]
```diff
@@ -53,34 +53,42 @@ which is supported by the tldr-pages project maintainers:
...
+see our [web client](https://tldr.ostera.io/)
```
Comment (waldyrious) [inline README.md:68]: I'd rather not call this "our" client, both because there are more than one ways to consume tldr-pages' content online, and because the tldr-pages project doesn't in fact manage that client. So maybe:

```suggestion
see the web client at <https://tldr.ostera.io>.
```

File: README.md (hunk 11 above)
Comment (waldyrious) [reply README.md:90]: I lost track of the line numbers now that you've made changes that affected them :sweat_smile: but IIRC, I was referring to the change "If you'd like to contribute to translations" → "For translations".

And yeah, I get your point, but I still prefer the sentence to be explicit and not depend on the reader reading the content of this file in full and in sequence.

File: README.md (hunk 10 above)
Comment (CleanMachine1) [reply README.md:88]:
```suggestion
If it's your first time contributing, have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
```

File: README.md (hunk 13 above)
Comment (CleanMachine1) [reply README.md:69]:
```suggestion
If you just want to browse without an internet connection or installing software,
checkout the [PDF version](https://tldr.sh/assets/tldr-book.pdf).

For browsing without installing a client to your computer,
see our [web client](https://tldr.ostera.io/)  

```

File: README.md (hunk 11 above)
Comment (CleanMachine1) [reply README.md:92]: I have also lost track of this. Could you please provide a suggestion?

File: README.md [hunk 18]
```diff
@@ -43,44 +42,55 @@ How about:
...
+```
 
-    pip3 install tldr
+Then you have direct access to simplified, easy-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+If you just want to browse without an internet connection or installing software,
+checkout the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
```
Comment (waldyrious) [inline README.md:67]:
```suggestion
check out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
```

File: README.md (hunk 11 above)
Comment (waldyrious) [reply README.md:92]: My suggestion is to just revert to the original:

```suggestion
If you'd like to contribute to translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
```

**Comment by waldyrious** [comment]
> Hey @waldyrious I'd like your opinion on this commit. 

Which one? f958f8a587?

**Comment by CleanMachine1** [comment]
My bad, yes https://github.com/tldr-pages/tldr/commit/f958f8a587442330571aa82a03162dd95bb5a2ca

I skipped over tagging it, apologies for that.

**Comment by waldyrious** [comment]
Regarding f958f8a, I get your argument that the relationship between the second part and the first one wasn't clear, but instead of removing the second part (which I think is important), I would prefer to clarify that relationship. For example:

> The source of all `tldr` pages are simple Markdown files in this repository,
> so you can edit them directly through the GitHub web interface
> and submit your changes as pull requests.

File: README.md [hunk 19]
```diff
@@ -43,44 +42,56 @@ How about:
...
-so you can edit them directly and submit your changes as pull requests.
+Some ways to contribute include:
+
+- Adding your favorite command which isn't covered.
+- Adding examples or improving the content of an existing page.
+- Translating pages into different languages.
+
+All `tldr` pages are written in markdown, so they can be edited fairly simply and changes can be submitted in
+pull requests here using git on the command-line or 
+using the GitHub web interface
```
Comment (waldyrious) [inline README.md:88]: I actually prefer my suggested wording to this one. Can you explain the reasoning g for the changes you made to it?

Comment (CleanMachine1) [reply README.md:88]: I didn't like using Git on the command line wasn't included however the first part of the sentence I don't mind, either works for me.

**Review by EmilySeville7cfg — APPROVED** [review]

File: README.md [hunk 20]
```diff
@@ -43,44 +42,56 @@ How about:
...
-so you can edit them directly and submit your changes as pull requests.
+Some ways to contribute include:
+
+- Adding your favorite command which isn't covered.
+- Adding examples or improving the content of an existing page.
+- Translating pages into different languages.
+
+All `tldr` pages are written in markdown, so they can be edited fairly simply and changes can be submitted in
+pull requests here using Git on the command-line or
+using the GitHub web interface
```
Comment (blueskyson) [inline README.md:88]:
```suggestion
using the GitHub web interface.
```

File: README.md [hunk 21]
```diff
@@ -43,44 +42,56 @@ How about:
...
+pip3 install tldr
+```
 
-    pip3 install tldr
+Then you have direct access to simplified, easy-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`.
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+If you just want to browse without an internet connection or installing software,
```
Comment (CleanMachine1) [inline README.md:66]:
```suggestion
If you just want to browse in the future, without an internet connection or installing software,
```
Open to other ideas here, the current sentence implies that the PDF can be downloaded without an internet connection.

Comment (blueskyson) [inline README.md:66]: My suggestion.
```suggestion
If you just want to browse without installing a tldr client,
```

Comment (blueskyson) [reply README.md:66]: My suggestion.
```suggestion
If you just want to browse without installing a tldr client,
```

Comment (258204) [reply README.md:66]:
```suggestion
If you want an offline version without installing any software,
```
I guess I'll join in too.

File: README.md [hunk 22]
```diff
@@ -43,44 +42,57 @@ How about:
...
 A popular and convenient way to access these pages on your computer
 is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-client),
 which is supported by the tldr-pages project maintainers:
 
-    npm install -g tldr
+```sh
+npm install -g tldr
+```
+
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
```
Comment (marchersimon) [inline README.md:58]:
```suggestion
Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip`.
```
Isn't `pip` an alias to `pip3` on most machines? And even if it isn't, wouldn't `tldr` still be installable with older versions?

File: README.md [hunk 23]
```diff
@@ -43,44 +42,57 @@ How about:
...
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+```sh
+pip3 install tldr
```
Comment (marchersimon) [inline README.md:61]:
```suggestion
pip install tldr
```

File: README.md [hunk 24]
```diff
@@ -43,44 +42,57 @@ How about:
...
+```
 
-    pip3 install tldr
+Then you have direct access to simplified, easy-to-read help for commands, such as tar,
```
Comment (marchersimon) [inline README.md:64]:
```suggestion
Then you have direct access to simplified, easy-to-read help for commands, such as `tar`,
```
Just for conformity

File: README.md (hunk 22 above)
Comment (CleanMachine1) [reply README.md:58]: not 100% on this but machines with python2, I think  `pip` refers to python2.
I can't tell if I am correct, please correct me if I am not.

File: README.md [hunk 25]
```diff
@@ -43,44 +42,57 @@ How about:
...
 
-All `tldr` pages are kept as Markdown files right here in this repository,
-so you can edit them directly and submit your changes as pull requests.
+Some ways to contribute include:
+
+- Adding your favorite command which isn't covered.
+- Adding examples or improving the content of an existing page.
+- Translating pages into different languages.
+
+All `tldr` pages are written in markdown, so they can be edited fairly simply and changes can be submitted in
```
[the remark this answers is not in the document]
Comment (CleanMachine1) [reply README.md:87]:
```suggestion
All `tldr` pages are written in markdown, so they can be edited fairly easily and changes can be submitted in
```

File: README.md (hunk 22 above)
Comment (marchersimon) [reply README.md:57]: But would machines with Python2 even be able to run pip3, which needs Python3?

Well, I think we could just keep pip3, since it doesn't really matter.

File: README.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (marchersimon) [reply README.md:86]: I'm not a big fan of the two adverbs in a row. It sounds a little like "Can be edited fairly & can be edited easily".

How about "rather easily" or "quite easily"?

[the remark this answers is not in the document]
Comment (CleanMachine1) [reply README.md:86]:
```suggestion
All `tldr` pages are written in markdown, so they can be edited quite easily and changes can be submitted in
```

**Review by marchersimon — APPROVED** [review]
Looks good, thanks
